tests/test_cli_compile.py

```diff
@@ -458,8 +458,32 @@ def test_quiet_option(runner):
     with open("requirements", "w"):
         pass
     out = runner.invoke(cli, ["--quiet", "-n", "requirements"])
-    # Pinned requirements result has not been written to output
-    assert "Dry-run, so nothing updated." == out.stderr.strip()
+    # Pinned requirements result has not been written to output. (An empty
+    # out.stderr results in a ValueError raised by Click)
+    try:
+        assert "" == out.stderr.strip()
+    except ValueError:
+        pass
```

Comment:
There will be no exception if `assert "" == out.stderr_bytes`

Reply:
Thanks for the suggestion!